src/Sylius/Behat/Context/Ui/Admin/ProductShowPageContext.php

```diff
@@ -432,7 +432,7 @@ public function iShouldSeeAttributeWithValueInLocale(string $attribute, string $
     }
 
     /**
-     * @Then I should see non-translatable attribute :attribute with value :value
+     * @Then /^I should see non-translatable attribute "([^"]+)" with value ([^"]+)%$/
```

Comment:
Why?

Reply:
The `%` inside `:value` was behaving weirdly